HLTrigger/Configuration/python/HLT_75e33/modules/hltTripleMuon3DZ1p0_cfi.py

```diff
@@ -4,8 +4,6 @@
     MaxDZ = cms.double(1.0),
     MinDR = cms.double(-1),
     MinN = cms.int32(3),
-    MinPixHitsForDZ = cms.int32(0),
-    checkSC = cms.bool(False),
```

Comment:
Why this removal?

Reply:
`HLT2L1TkMuonL1TkMuonDZ` does not use those parameters, even pre-PR [*]. The PR cleans the `fillDescriptions` of this instantiation accordingly, making those extra parameters invalid.

[*] It [reads them from the PSet](https://github.com/cms-sw/cmssw/blob/3fa9bc2b0cc042ccc096cb794d35a25f227c3529/HLTrigger/HLTfilters/plugins/HLTDoubletDZ.cc#L147), but those does not use them in [`HLTDoubletDZ<l1t::TrackerMuon, l1t::TrackerMuon>::hltFilter`](https://github.com/cms-sw/cmssw/blob/3fa9bc2b0cc042ccc096cb794d35a25f227c3529/HLTrigger/HLTfilters/plugins/HLTDoubletDZ.cc#L518) nor [`HLTDoubletDZ<l1t::TrackerMuon, l1t::TrackerMuon>::computeDZ`](https://github.com/cms-sw/cmssw/blob/3fa9bc2b0cc042ccc096cb794d35a25f227c3529/HLTrigger/HLTfilters/plugins/HLTDoubletDZ.cc#L433).